herddb-core/src/main/java/herddb/model/planner/LimitOp.java

```diff
@@ -118,4 +118,9 @@ public PlannerOp optimize() {
         }
         return this;
     }
+
+    @Override
+    public String toString() {
+        return "LimitOp{value = " + this.maxRows.toString() + " }";
```

Comment:
We should recursively call testing on the input op

Reply:
Will do.